addons/calendar/static/src/js/calendar_view.js

```diff
@@ -30,6 +29,46 @@ var AttendeeCalendarPopover = CalendarPopover.extend({
         }
     },
 
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+    /**
+     * @return {boolean}
+     */
+    isCurrentPartnerAttendee() {
+        return this.event.record.partner_ids.includes(session.partner_id);
+    },
+    /**
+     * @override
+     * @return {boolean}
+     */
+    isEventDeletable() {
+        return this._super() && (this._isEventPrivate() ? this.isCurrentPartnerAttendee() : true);
+    },
+    /**
+     * @return {boolean}
+     */
+    isEventDetailsVisible() {
+        return this._isEventPrivate() ? this.isCurrentPartnerAttendee() : true;
+    },
+    /**
+     * @return {boolean}
+     */
+    isEventEditable() {
+        return this._isEventPrivate() ? this.isCurrentPartnerAttendee() : true;
+    },
```

Comment:
I would say both methods should be overrides from superclass, and superclass just return `true`.

Reply:
Done!